installinator/src/dispatch.rs

```diff
@@ -151,9 +127,19 @@ struct InstallOpts {
     artifact_ids: ArtifactIdOpts,
 
     // TODO: checksum?
+    /// Install on a gimlet's M.2 drives, found via scanning for hardware.
+    ///
+    /// WARNING: This will overwrite the boot image slice of both M.2 drives, if
+    /// present!
+    #[clap(long)]
+    install_on_gimlet: bool,
 
     // The destination to write to.
-    destination: Utf8PathBuf,
+    #[clap(
+        required_unless_present = "install_on_gimlet",
+        conflicts_with = "install_on_gimlet"
+    )]
+    destination: Option<Utf8PathBuf>,
```

Comment:
I could go either way; I left it this way to keep the existing CLI working as-is, but since you're probably the only person who's used it, if you like making this an option better I'm happy to do that.

Reply:
(I'm going to go ahead and merge this PR so we don't wait for CI on a tiny change; if we want to `clap(long)` this we can do it in any followup.)